src/main/java/com/adaptavist/tm4j/jenkins/extensions/buildsteps/FeatureFilesDownloader.java

```diff
@@ -45,25 +43,24 @@ public FeatureFilesDownloader(String serverAddress, String projectKey, String ta
     }
 
     @Override
-    public void perform(Run<?, ?> run, FilePath workspace, Launcher launcher, TaskListener listener) {
+    public void perform(@Nonnull Run<?, ?> run,@Nonnull FilePath workspace,@Nonnull Launcher launcher, TaskListener listener) {
         final PrintStream logger = listener.getLogger();
         logger.printf("%s Downloading feature files...%n", INFO);
-        List<JiraInstance> jiraInstances = getDescriptor().getJiraInstances();
+        List<Instance> jiraInstances = getDescriptor().getJiraInstances();
         try {
             new Validator().validateProjectKey(this.projectKey)
                     .validateTargetPath(this.targetPath)
                     .validateServerAddress(this.serverAddress);
             String tql = format("testCase.projectKey = '%s'", this.projectKey);
             Tm4jJiraRestClient tm4jJiraRestClient = new Tm4jJiraRestClient(jiraInstances, serverAddress);
-            tm4jJiraRestClient.exportFeatureFiles(run.getRootDir(), workspace, targetPath, tql, logger);
+            tm4jJiraRestClient.importFeatureFiles(run.getRootDir(), workspace, targetPath, tql, logger);
```

Comment:
Nice! `tm4jJiraRestClient` is actually importing feature files.

Reply:
As per my previous comment, what if we stick to the naming convention we have for the build steps title? "downloadFeatureFiles". I think that would be clearer.